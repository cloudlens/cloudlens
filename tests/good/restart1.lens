stream (e in log.a)  {
       print(1)
}

stream {
       print(e.message)
}

restart log.a

stream (e in log.a)  {
       print(2)
}

restart log

stream {
       print(e.message)
}

