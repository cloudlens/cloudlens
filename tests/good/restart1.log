[
  {"message" : "foo 1", "a" : [1,2,3,4]},
  {"message" : "foo 2", "a" : [1,2,3,4]},
  {"message" : "foo 3", "a" : [1,2,3,4]},
  {"message" : "foo 4", "a" : [1,2,3,4]}
]