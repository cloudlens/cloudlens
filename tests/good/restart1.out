1
1
1
1
foo 1
2
2
2
2
1
1
1
1
foo 2
2
2
2
2
1
1
1
1
foo 3
2
2
2
2
1
1
1
1
foo 4
2
2
2
2
foo 1
foo 2
foo 3
foo 4
